selenoid/selenoid.go

```diff
@@ -112,9 +121,61 @@ func Status(ctx context.Context, baseUrl string) ([]byte, error) {
 		return nil, err
 	}
 
+
+	req, err = http.NewRequest("GET", baseUrl+videosPath+"?json", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	_ = httpDo(ctx, req.WithContext(ctx), func(resp *http.Response, err error) error {
+		if err != nil {
+			return err
+		}
+		defer resp.Body.Close()
+		return json.NewDecoder(resp.Body).Decode(&videos)
+	})
+
+	state.Videos = videos
+
 	return json.Marshal(toUI(state, baseUrl))
 }
 
+
+func Video(orig *http.Request, baseUrl string, video string) ([]byte,*http.Response,error) {
+	var respVideo []byte
+
+	r:= &http.Response{}
+
+	req, err := http.NewRequest("GET", baseUrl+videosPath+"/"+video, nil)
```

Comment:
Please use `httputil.ReverseProxy` instead of proxying bytes manually, e.g.:
```
                u, err := url.Parse(baseUrl+videosPath+"/"+video)
                if err != nil {
                        // Handler error
               }
		(&httputil.ReverseProxy{
			Director: func(r *http.Request) {
				r.URL = u
			}
		}).ServeHTTP(w, r)
```

Reply:
implemented in last commit